qtopconf.yaml

```diff
@@ -218,7 +218,8 @@ workernodes_matrix:
 #Yellow
 
 extract_info:
-  sourcefile: cat -E $HOME/.local/qtop/getent_passwd.txt # this should print an /etc/passwd file
+  user_details1: cat -E $HOME/.local/qtop/getent_passwd.txt # this should print an /etc/passwd file
+  user_details2: getent passwd %s # alternative method to get details should the first one fail
```

Comment:
This PR is good and has proven its value; however a minor change:
* user_details1 -> user_details_cache
* user_details2 -> user_details_realtime
tune comments accordingly!